src/_pyinstaller_hooks_contrib/hooks/stdhooks/hook-pypemicro.py

```diff
@@ -0,0 +1,43 @@
+# ------------------------------------------------------------------
+# Copyright (c) 2022 PyInstaller Development Team.
+#
+# This file is distributed under the terms of the GNU General Public
+# License (version 2.0 or later).
+#
+# The full license is available in LICENSE.GPL.txt, distributed with
+# this software.
+#
+# SPDX-License-Identifier: GPL-2.0-or-later
+# ------------------------------------------------------------------
+
+# Hook for the pypemicro module: https://github.com/nxpmicro/pypemicro
+
+import os
+from PyInstaller.utils.hooks import get_package_paths, is_module_satisfies
+from PyInstaller.log import logger
+from PyInstaller.compat import is_darwin
+
+binaries = list()
+if is_module_satisfies('pyinstaller >= 5.0'):
+    from PyInstaller import isolated
+
+    @isolated.decorate
+    def get_safe_libs():
+        from pypemicro import PyPemicro
+        libs = PyPemicro.get_pemicro_lib_list()
+        return libs
+
+    pkg_base, pkg_dir = get_package_paths("pypemicro")
+    for lib in get_safe_libs():
+        source_path = lib['path']
+        source_name = lib['name']
+        dest = os.path.relpath(source_path, os.path.dirname(pkg_base))
```

Comment:
I think this should be using `pkg_base` instead of `os.path.dirname(pkg_base))`. 

Both work as expected, but that's only because `pkg_base`  has a trailing separator, and so `os.path.dirname(pkg_base)` still returns the path to the site-packages directory. So while it works, it gives incorrect impression that we are after the parent directory, which we are not..

Reply:
you are right, it is incorrect, probably leftover from the `remove_prefix`